spec/requests/network_routers_spec.rb

```diff
@@ -48,4 +48,70 @@
       expect(response).to have_http_status(:forbidden)
     end
   end
+
+  describe "DELETE /api/network_routers" do
+    it "can delete a router" do
+      network_router = FactoryGirl.create(:network_router)
+
```

Comment:
nit, but can you remove this blank line here? This way all of the setup lines go together for the spec. Same for other ones